roles/upload_helm_chart/tasks/main.yml

```diff
@@ -1,12 +1,7 @@
-- name: Upload Helm chart
-  run_once: true
-  # NOTE(mnaser): https://github.com/ansible-collections/ansible.posix/issues/381
-  #               https://github.com/ansible-collections/ansible.posix/pull/433
-  vexxhost.kubernetes.synchronize:
-    src: "{{ upload_helm_chart_src }}"
-    dest: "{{ upload_helm_chart_dest }}"
-    archive: false
-    recursive: true
-    checksum: true
-    delete: true
-    use_ssh_args: true
+- name: Upload helm char with synchronize
+  ansible.builtin.import_tasks: synchronize.yml
```

Comment:
If we make this out to be `{{ upload_helm_chart_method }}.yml` would this save us from skipping a whole bunch of tasks?

Reply:
Done